hstrat/phylogenetic_inference/tree/_impl/_build_tree_searchtable_cpp_native.py

```diff
@@ -0,0 +1,9 @@
+import re
+
+from ...._auxiliary_lib import import_cpp_impls
+
+_impl_mod = import_cpp_impls(
+    "_build_tree_searchtable_cpp_impl",
+    re.sub(r"\.[a-za-z0-9_]+$", "", __name__),
```

Comment:
should one of the a-z be capitalized? and why do we need the regex?

Reply:
add a comment about it or maybe even make it a helper function if we do need the regex